package.json

```diff
@@ -9,12 +9,14 @@
   "scripts": {
     "start": "wp-scripts start",
     "build": "wp-scripts build --mode production",
+    "build:preact": "wp-scripts build --mode production --config webpack.preact.config.js --webpack-no-externals",
     "lint:css": "wp-scripts lint-style",
     "setupgithooks": "bin/copy-git-hooks.sh",
     "test": "env JEST_TIMEOUT=15000 WP_USERNAME=admin WP_PASSWORD=admin WP_BASE_URL=https://www.planet4.test wp-scripts test-e2e",
     "test:watch": "env JEST_TIMEOUT=30000 PUPPETEER_HEADLESS=false PUPPETEER_SLOWMO=100 WP_USERNAME=admin WP_PASSWORD=admin WP_BASE_URL=https://www.planet4.test wp-scripts test-e2e"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-decorators": "^7.14.2",
```

Comment:
Could this be a leftover from the first PR? I didn't see anything added that uses this.